bridges/FeedReducerBridge.php

```diff
@@ -54,7 +54,7 @@ public function getItems(){
 	}
 
 	public function getName(){
-		$trimmedPercentage = preg_replace('/[^0-9]/', '', $this->getInput('percentage'));
+		$trimmedPercentage = preg_replace('/[^0-9]/', '', $this->getInput('percentage') || '');
```

Comment:
This operation returns a bool in php.

Reply:
Bummer, too much Python I guess. Can I change the `getInput` function to take a default?

Reply:
Let's check if the tests pass...

Reply:
Can do `$this->getInput('percentage') ?? ''`

Reply:
That's much simpler. Thanks!